javascript/script.js

```diff
@@ -8,18 +8,19 @@ let newItemInput;
 
 const $ = id => document.getElementById(id);
 const setToDoList = list => toDoList = list;
-
+let searchInput = $("search-text");
+const validate = () => {
+	//modal display on empty input
+	if (newItemInput.value.trim() === "") {
+		showEmptyInputModal();
+		return true;
+	}
+}
 //Function to add item to to-do list
 function addToDoItem(toDoItem) {
```

Comment:
Covert everything to arrow functions. Be careful to cover everything. This is the third time I'm requesting this change.